code/modules/projectiles/projectile.dm

```diff
@@ -543,6 +543,8 @@
 	else
 		var/mob/living/L = target
 		if(!direct_target)
+			if(!initial(L.density)) //If they're always non-dense (e.g. mice), the projectile can pass over

```

Comment:
this makes revenants unable to be hit

Reply:
like, always

Reply:
Yeah I was gonna say... Plus, this messes up anything that has density change, which I'm fairly sure rev does.